src/main/resources/META-INF/rewrite/was-to-liberty.yml

```
@@ -0,0 +1,41 @@
+#
+# Copyright 2023 the original author or authors.
+# <p>
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# <p>
+# https://www.apache.org/licenses/LICENSE-2.0
+# <p>
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+########################################################################################################################
+# WebSphere traditional to Liberty
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.liberty
+displayName: Migrate from WebSphere traditional to Liberty
+description: >
+  Use this category of rules to identify code changes needed when migrating
+  from WebSphere Application Server traditional to Liberty.
+recipeList:
+ - org.openrewrite.java.liberty.WebSphereUnavailableSSOCookieMethod
+ - org.openrewrite.java.liberty.WebSphereUnavailableSSOTokenMethod
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.liberty.WebSphereUnavailableSSOCookieMethod
+recipeList:
+  - org.openrewrite.java.liberty.ChangeMethodObjectInvocation
```

If my below recipe suggestion is adopted, then we can likely replace this block with a reference there.
```suggestion
 - org.openrewrite.java.liberty.RevokeSSOCookiesToLogout
```